Sources/Markdown/Markdown.docc/Markdown/BlockDirectives.md

```diff
@@ -150,31 +150,4 @@ Pass the `.parseBlockDirectives` option when parsing a document to enable block
 let document = Document(parsing: source, options: .parseBlockDirectives)
 ```
 
-## Collecting Diagnostics
```

Comment:
Looks like ``Diagnostic``, ``DiagnosticEngine``, and ``DiagnosticConsumer`` are no longer exist. But should we keep the "Collecting Diagnostics" and update its content to something like DirectiveArgumentText.ParseError?

Reply:
I think updating the full code example is out-of-scope for this PR but would be good to address in a follow-up.

I think you're right though that we should keep the section so I added back a minimal version of it here: https://github.com/apple/swift-markdown/pull/32/commits/de0e39db201ff735fd0500fb936bad212dc83daa